src/components/data-service-form/index.tsx

```diff
@@ -734,8 +734,8 @@ const DataServiceForm = ({
                   name='mediaTypes'
                   labelText={
                     isReadOnly
-                      ? 'Registrerte mediatyper'
-                      : 'Velg blant registrerte mediatyper'
+                      ? 'Registrerte medietyper'
+                      : 'Velg blant registrerte medietyper'
```

Comment:
Foreslår at vi har tilsvarende tekst som for format:
"Søk på og velg blant registrerte mediatyper"

![image](https://github.com/user-attachments/assets/bb377584-380a-4a4a-ae44-dfdde931a334)